src/routes/did.rs

```diff
@@ -220,7 +220,7 @@ async fn login_with_did(
                 oidc_context.redirect_uri.clone(),
                 id_token,
                 refresh_token,
-                oidc_context.state.clone(),
+                oidc_context.state.unwrap_or_default().clone(),
```

Comment:
So is it fine to return an empty value for the `state` parameter? Or should it be omitted in case `state` is `None`?

Reply:
I think it's fine, but I removed it anyways 43bdb84828cb